docs/build_tutorial.md

```diff
@@ -1,38 +1,79 @@
 # How to build Holochain DNA
 
-*as of 12-08-2020*
+*as of 28-10-2020*
 
-## Important note about dependencies
+## Steps
 
-Effective zome development requires a few cargo dependencies which are not yet published to crates.io and exist only in the private [holochain-rsm GitHub repo](https://github.com/holochain/holochain). Since the repo is private, we cannot simply point our Cargo.toml file directly to it without some configuration. There are two options:
+### 0. Build `holochain` and `dna-util`
 
-1. Clone the Holochain repo, and use local paths in your Cargo.toml(s) to refer to your local checkout of Holochain, or
-2. If you use SSH keys for GitHub, configure your ssh-agent so that Cargo can be aware of your key, per https://doc.rust-lang.org/cargo/appendix/git-authentication.html. Then you can use remote git deps in your zomes' Cargo.toml(s).
+You'll need two binaries on your PATH to develop DNAs: the actual Holochain (`holochain`) conductor binary, and the dna-util library which assists with assembling Wasms into a DNA file.
 
-At the time of writing, I could not get the remote option to work, so the example zomes here use local paths (option 1).
+There are two ways you can approach that, via a [nix-shell](https://nixos.org/manual/nix/stable/#ch-installing-binary) which handles the majority for you, or via direct Rust installation to your computer. Instructions for both follow.
 
-## Steps
+#### nix-shell
 
-### 0. Build `holochain` and `dna-util`
+Install nix, **linux**
+```bash
+sh <(curl -L https://nixos.org/nix/install) --no-daemon
+```
+Install nix, **macOS**
+```bash
+sh <(curl -L https://nixos.org/nix/install) --darwin-use-unencrypted-nix-store-volume
+```
 
-You'll need two binaries to develop DNAs: the actual Holochain conductor binary, and the dna-util library which assists with assembling Wasms into a DNA file.
+Clone the holochain/holochain repo
+```bash
+git clone git@github.com:holochain/holochain.git
+```
 
-- Clone the repo: `git clone https://github.com/holochain/holochain && cd ./holochain`
-- Install conductor binary: `cargo install --path crates/holochain`
-- Install dna-util binary: `cargo install --path crates/dna_util`
+Enter the directory
+```bash
+cd holochain
+```
+
+Launch a nix-shell, based on holochain/holochain's nix-shell configuration
+```bash
+nix-shell
+```
+
+Install the `holochain` and `dna-util` binaries using the built-in installer
+```bash
+hc-install
+```
+
+Confirm that they are there by running `holochain -V` and `dna-util -V`, and that you see simple version number outputs.
+
+#### native rust
+
+Install Rust
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Switch to the proper Rust "toolchain" that Holochain needs
+```bash
+rustup install stable-2020-08-03
+rustup default stable-2020-08-03
```

Comment:
Is there any reason to pin the rust version in this doc? Won't it just go out of date? We should be able to always support the latest stable and if not that's a bug

Reply:
This note in the `core_testing.md` doc provides the reference that demonstrates why you can't use latest stable :s
https://github.com/holochain/holochain/blob/develop/docs/core_testing.md#requirements